helm/odigos/templates/instrumentor/webhooks.yaml

```diff
@@ -114,6 +107,4 @@ metadata:
   annotations:
     "helm.sh/hook": "pre-install,pre-upgrade"
     "helm.sh/hook-delete-policy": "before-hook-creation"
```

Comment:
are these relevant after the change? 
as it is now managed by the controller, I am not sure if these instructions to helm about creation and deletion are actually needed.